alacritty_terminal/src/term/mod.rs

```diff
@@ -50,12 +50,26 @@ pub const MIN_SCREEN_LINES: usize = 1;
 
 /// Cursor storing all information relevant for rendering.
 #[derive(Debug, Eq, PartialEq, Copy, Clone, Deserialize)]
-struct RenderableCursor {
+pub struct RenderableCursor {
+    color: Rgb,
```

Comment:
Decided to store here just to avoid creating one more struct, but it makes certain things a bit more uglier, I guess.